Sources/TuistKit/Generator/GeneratorFactory.swift

```diff
@@ -18,7 +18,7 @@ protocol GeneratorFactorying {
     /// - Parameter ignoreCache: True to not include binaries from the cache.
     /// - Returns: The generator for focused projects.
     func focus(config: Config,
-               sources: Set<String>,
+               sources: Set<String>?,
```

Comment:
Could you remember me when it can be empty? If never, we can use the empty set instead of `nil` to request to focus on internal targets